### PR TITLE
Keep list of pending transactions up to date - Closes #140

### LINKED ISSUE #140: Keep list of pending transactions up-to-date
### Description

The list of pending transactions changes every block is forged in the network. Make sure the list is retrieved and stored in the local in-memory database every time new block arrives.

### Motivation

- There is a UI requirement to support pending transaction list

### Acceptance Criteria

- Pending transactions need to be available and the list has to be up-to-date
- List of transactions has to be refreshed in short intervals (1s)

### PR DESCRIPTION
**Pull request by nagdahimanshu**
### What was the problem?
This PR resolves #140 

### How was it solved?

- [x] Add new route /transactions/pending
- [x] Pending transactions need to be available and the list has to be up-to-date
- [x] List of transactions has to be refreshed in short intervals (1s)

### How was it tested?
Local

### CODE REVIEW

**Review by sameersubudhi — APPROVED** [review]
LGTM 👍

File: services/core/methods/controllers/transactions.js [hunk 1]
```diff
@@ -158,10 +158,20 @@ const getTransactionsStatistics = async ({
 const getTransactionsStatisticsDay = (params) => getTransactionsStatistics({ aggregateBy: 'day', ...params });
 const getTransactionsStatisticsMonth = (params) => getTransactionsStatistics({ aggregateBy: 'month', ...params });
 
+const getPendingTransactions = async (params) => {
+	const result = await CoreService.getPendingTransactions(params);
+	return {
+		data: result.data,
+		meta: result.meta,
+		links: result.links,
```
Comment (MichalTuleja) [inline services/core/methods/controllers/transactions.js:166]: Links are not needed here.

File: services/core/shared/core/transactions.js [hunk 2]
```diff
@@ -91,8 +92,8 @@ const getTransactions = async params => {
 			const dbResult = await db.find(inputData);
 			if (dbResult.length > 0) {
 				const latestBlock = (await getBlocks({ limit: 1 })).data[0];
-				dbResult.map(tx => {
-					tx.confirmations = latestBlock.confirmations + latestBlock.height - tx.height;
+				dbResult.map((tx) => {
+					tx.confirmations =						latestBlock.confirmations + latestBlock.height - tx.height;
```
Comment (MichalTuleja) [inline services/core/shared/core/transactions.js:96]: Indentation?

**Review by MichalTuleja — CHANGES_REQUESTED** [review]

File: services/core/methods/controllers/transactions.js (hunk 1 above)
Comment (nagdahimanshu) [reply services/core/methods/controllers/transactions.js]: Removed!

File: services/core/shared/core/transactions.js (hunk 2 above)
Comment (nagdahimanshu) [reply services/core/shared/core/transactions.js]: Fixed indentation.

File: services/core/shared/core/transactions.js [hunk 3]
```diff
@@ -22,7 +23,7 @@ const { getBlocks } = require('./blocks');
 
 const logger = Logger();
 
-const formatSortString = sortString => {
+const formatSortString = (sortString) => {
```
Comment (MichalTuleja) [inline services/core/shared/core/transactions.js:26]: Please update formatting

File: services/core/shared/core/transactions.js [hunk 4]
```diff
@@ -73,15 +74,15 @@ const getSelector = (params) => {
 	return result;
 };
 
-const getTransactions = async params => {
+const getTransactions = async (params) => {
```
Comment (MichalTuleja) [inline services/core/shared/core/transactions.js:77]: Please update formatting

File: services/core/shared/core/transactions.js [hunk 5]
```diff
@@ -91,7 +92,7 @@ const getTransactions = async params => {
 			const dbResult = await db.find(inputData);
 			if (dbResult.length > 0) {
 				const latestBlock = (await getBlocks({ limit: 1 })).data[0];
-				dbResult.map(tx => {
+				dbResult.map((tx) => {
```
Comment (MichalTuleja) [inline services/core/shared/core/transactions.js:95]: Please update formatting

**Review by MichalTuleja — CHANGES_REQUESTED** [review]

**Review by MichalTuleja — APPROVED** [review]

**Pull request merged**